workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -0,0 +1,116 @@
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Tools
+{
+    public static class LocalLaunch
+    {
+        private static readonly string
+            SpatialProjectRootDir = Path.GetFullPath(Path.Combine(Application.dataPath, "..", "..", ".."));
+
+        // Windows: The exit code is 0xc000013a when the user closes the console window, or presses Ctrl+C.
+        private const int WindowsCtrlCExitCode = -1073741510;
+
+        // Unixy: The exit code is 128 + SIGINT (2).
```

Comment:
Typo: "Unix"

Reply:
Its a portmanteau - "Unity" + "Unix" 😉

Reply:
:smooth:

Reply:
Hehe, it was actually meant to mean "Unix-like", I'll just change it to that.